### PR TITLE
Added auto-reload during development (optional)

### PR DESCRIPTION
**Pull request by MelodicCrypter**
Hi @neolao . Just an optional enhancement. Just want to add auto-reload during development. But this is not a priority.

### CODE REVIEW

**Comment by neolao** [comment]
It's a bit annoying to run 2 commands. 

I suggest to add also https://www.npmjs.com/package/concurrently

in order to run only one command:
```json
"start:auto-mac": "concurrently --kill-others \"npm run dev\" \"export AUTO_RELOAD=true && webpack --mode development && electron .\"",
```

File: index.js [hunk 1]
```diff
@@ -4,6 +4,11 @@ const fs = require("fs");
 const configYaml = require("config-yaml");
 const isDev = require("electron-is-dev");
 
+// Enable auto-reload in development if set
+try {
+  if (process.env.AUTO_RELOAD) require('electron-reloader')(module);
```
Comment (neolao) [inline index.js]: I suggest to also add `isDev` if it's only in development.

```suggestion
  if (isDev && process.env.AUTO_RELOAD) require('electron-reloader')(module);
```

**Comment by MelodicCrypter** [comment]
Nice. Okay, I will modify tomorrow. Thanks! 👍

**Comment by MelodicCrypter** [comment]
@neolao already modified.

**Comment by neolao** [comment]
seems good

**Pull request merged**